test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/LanguageServiceTests.cs

```diff
@@ -290,6 +280,44 @@ public void AutoCompleteFindCompletions()
             Assert.True(completions.Length > 0);
         }
 
+        /// <summary>
+        /// Verify that GetSignatureHelp returns not null when the provided TextDocumentPosition
+        /// has an associated ScriptParseInfo and the provided query has a function that should
+        /// provide signature help.
+        /// </summary>
+        [Fact]
+        public async void GetSignatureHelpReturnsNotNullIfParseInfoInitialized()
+        {
+            // When we make a connection to a live database
+            ScriptFile scriptFile;
+            ConnectionInfo connInfo = TestObjects.InitLiveConnectionInfo(out scriptFile);
+
+            // And we place the cursor after a function that should prompt for signature help
+            string queryWithFunction = "EXEC sys.fn_isrolemember ";
+            scriptFile.Contents = queryWithFunction;
+            TextDocumentPosition textDocument = new TextDocumentPosition
+            {
+                TextDocument = new TextDocumentIdentifier
+                {
+                    Uri = scriptFile.ClientFilePath
+                },
+                Position = new Position
+                {
+                    Line = 1,
```

Comment:
Could you verify if Position.Line should be 0-based?  I think the SqlParser methods are 1-based and most every thing else is 0-based if I recall correctly?